vast/integration/vast_integration_suite.yaml

```diff
@@ -592,16 +599,22 @@ tests:
     tags: [server, client, import-export, transforms]
     fixture: ServerTester
     steps:
-      - command: "import -b suricata 'pass
-                               | select timestamp, flow_id, src_ip, dest_ip, src_port
-                               | drop timestamp
-                               | hash --salt=\"abcdefghij12\" flow_id
-                               | drop flow_id
-                               | pseudonymize -m \"crypto-pan\" -s \"123456abcdef\" src_ip, dest_ip
-                               | extend abc=123,abc_str=\"123\", abc_list=[1,2,3]
-                               | rename new_list=abc_list
-                               | replace new_list=2
-                               | where #type ==\"suricata.alert\" || #type == \"suricata.fileinfo\"'"
+      - command: |
```

Comment:
Can there be an integration test that checks and confirms the intricacies of the multi-line comment syntax (i.e. invalid/valid syntax)?
e.g.
* `select timestamp | /**/`
* `select timestamp /*double ending*/ slash*/`
* `select timestamp /*double beginning /* is valid */`

Reply:
Implemented exactly these three tests in d4c0e83fe.